hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -36,19 +36,59 @@
 
     private boolean splitBrainProtectionEnabled;
     private final List<ResourceConfig> resourceConfigs = new ArrayList<>();
+    private boolean autoRestartEnabled = true;
 
     /**
-     * Returns true if split brain protection is enabled
+     * Returns true if {@link #setSplitBrainProtection(boolean)} is enabled.
      */
     public boolean isSplitBrainProtectionEnabled() {
         return splitBrainProtectionEnabled;
     }
 
     /**
-     * Sets split brain protection configuration
+     * Configures the split brain protection feature. When enabled, the job will only
+     * be restarted after a topology change if the quorum size can be met.
+     * The quorum size is calculated as {@code cluster size at job submission time / 2 + 1}.
```

Comment:
The net length of a text line should not exceed 72 chars (the initial indentation and `*` are not counted).